src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -70,6 +70,8 @@ SocketStreamerAdapter::SocketStreamer::SocketStreamer(
     ip_(ip),
     port_(port),
     header_(header),
+    server_socket_(),
```

Comment:
I mean it will be anyway initialized in appropriate order. But any way it's not a subject for change